src/factories/custom/Restricted_PIM_Factory_v1.sol

```diff
@@ -250,6 +254,20 @@ contract Restricted_PIM_Factory_v1 is
         // get bonding curve / funding manager
         address fundingManager =
             address(deployedContracts.orchestrator.fundingManager());
+        // get payment router
+        address[] memory modules = deployedContracts.orchestrator.listModules();
+        address paymentRouter;
+        bytes4 paymentRouterInterfaceId =
+            type(ILM_PC_PaymentRouter_v1).interfaceId;
+        for (uint i = 0; i < modules.length; i++) {
```

Comment:
Slightly gas-optimized way. I know we are a bit all over the place in our codebase but this is the one we usually like to use. 

```suggestion
        for (uint i; i < modules.length; ++i) {
```

(Removed double initialization of `i` and `++i` being slightly cheaper than `i++`)